test/gtest/ucp/test_ucp_am.cc

```diff
@@ -29,10 +29,10 @@ extern "C" {
 #define UCP_REPLY_ID 1
 #define UCP_RELEASE 1
 
-class test_ucp_am_base : public ucp_test {
-protected:
-    using variant_vec_t = std::vector<ucp_test_variant>;
+using variant_vec_t = std::vector<ucp_test_variant>;
```

Comment:
can we move it to ucp_test class?

Reply:
Yes, we can.
Many files should be changed then.
I think it is better to do in separate review.

Reply:
Can we move it there but not change existing places yet?